rssfeed/models.py

```diff
@@ -17,37 +18,27 @@ def __unicode__(self):
         return self.title or self.xml_url
 
     def save(self, *args, **kwargs):
-        """Poll new Feed"""
+        """Poll new Feed """
         try:
             Feed.objects.get(xml_url=self.xml_url)
             super(Feed, self).save(*args, **kwargs)
         except Feed.DoesNotExist:
             super(Feed, self).save(*args, **kwargs)
             from rssfeed.utils import poll_feed
```

Comment:
@cobusc Moving this import to the top of the file breaks the build, since the `poll_feed` function uses the `Entry` model, which hasn't been defined yet at that point, so I get `ImportError: cannot import name Entry`.